docs/spec/channel.md

```diff
@@ -8,7 +8,8 @@ different Channel CRDs implementations (e.g.`InMemoryChannel`) inside of in the
 the chosen _default_ `Channel` implementation, like the `InMemoryChannel`.
 
 A _channel_ logically receives events on its input domain and forwards them to
-its subscribers. Below is a specification for the generic parts of each
+its subscribers, the _initial_ event. The channel is able to receive _reply events_
```

Comment:
I wonder if _initial_ is the right word here. I'm not sure it's clear that for each incoming (or received) event the subscriber can reply with zero or one event. What do you think?